tempesta_fw/http_msg.c

```diff
@@ -475,21 +475,65 @@ __hdr_sub(TfwHttpMsg *hm, char *name, size_t n_len, char *val, size_t v_len,
 			{ .ptr = name,	.len = n_len },
 			{ .ptr = ": ",	.len = 2 },
 			{ .ptr = val,	.len = v_len },
-			{ .ptr = "\r\n", .len = 2 }
+			{ .ptr = "\r\n",.len = 2 } /* may be adjusted */
 		},
 		.len = n_len + v_len + 4,
 		.flags = 4 << TFW_STR_CN_SHIFT
 	};
+	TfwStr *hlc = TFW_STR_LAST(&hdr);
```

Comment:
Perhaps it's better to use just `&hdr[3]` or something like that as that is calculated at compile time versus the run time.